1.9/api/index.md

```diff
@@ -37,7 +37,7 @@ To determine the URL of your cluster, see [Cluster Access](/docs/1.9/api/access/
 - To determine the full URL of a API resource through a **proxy route**, join the cluster URL, route path, and backend component resource path.
 
     ```bash
-    ${cluster-url}/${route}/${component-resource-path}
+    ${cluster-url}/${route}/${resource-path}
```

Comment:
`${variable}` is bash string interpolation format. But I can change to `<variable>` if we want to be more consistent.